openshift/templates/patroni.yaml

```diff
@@ -340,9 +362,16 @@ parameters:
   - description: Version of Patroni image to be used.
     displayName: Patroni image version
     name: IMAGE_VERSION
-    value: "v10-build5"
+    value: "v10-pr-454"
```

Comment:
I'm now super confused about the whole patroni image versioning process, but just wanted to flag for you in case this is unintentional that there's a PR-specific value here

Reply:
You're not alone! I have myself confused - my though patterns evolved as such:
- Just having 10, becomes a pain, since my PR may be intricately linked to the version of the image. Also there's no context as to what 10 is.
- Using a build number is a pain, because it keeps going up, so I keep needing to change which version I use - again there's no context.
- Giving it a unique name helps me manage the image for my PR, and I know that when my PR goes to prod it will be associated to the correct image.
- Using a PR gives me context as to where this build came from, but may be confusing as subsequent PR's are going to be referring the the last PR that change the image-stream being used.

I'm open to suggestions.
Maybe v10-rev1, v10-rev2 etc. rather than PR? Will lose context but so what?

Reply:
I'm perfectly happy with using PR numbers as part of the naming convention. Just wanted to double-check that this is now the process and that it wasn't left over from you experimenting with a specific image just for the PR

Reply:
"process" is maybe too generous at this moment in time.